addons/stock_picking_wave/__openerp__.py

```diff
@@ -13,13 +13,13 @@
     'website': 'https://www.odoo.com/page/warehouse',
     'depends': ['stock'],
     'data': ['security/ir.model.access.csv',
-            'stock_picking_wave_view.xml',
-            'stock_picking_wave_data.xml',
-            'stock_picking_wave_sequence.xml',
+            'views/stock_picking_wave_view.xml',
```

Comment:
file should be named as stock_picking_wave_views.xml(do the same change for wizard/picking_to_wave_view.xml)